### PR TITLE
Removing Reprocess Lambda

### PR DESCRIPTION
**Pull request by codyseibert**
- we no longer write failed ES index calls to dynamo; instead, our stream will continuously retry any failed writes

### CODE REVIEW

**Review by rachaelparris — APPROVED** [review]
Nice! You can also delete `shared/src/persistence/dynamo/elasticsearch/getRecord.js` and its test, `shared/src/persistence/dynamo/elasticsearch/createElasticsearchReindexRecord.js` and its test, and `shared/src/persistence/dynamo/elasticsearch/deleteElasticsearchReindexRecord.js` and its test.

**Comment by rachaelparris** [comment]
> Nice! You can also delete `shared/src/persistence/dynamo/elasticsearch/getRecord.js` and its test, `shared/src/persistence/dynamo/elasticsearch/createElasticsearchReindexRecord.js` and its test, and `shared/src/persistence/dynamo/elasticsearch/deleteElasticsearchReindexRecord.js` and its test.

Oh, annnnd `shared/src/persistence/elasticsearch/indexRecord.js` and its test!

**Review by sutt0n — APPROVED** [review]

File: .nova/Configuration.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "index.ignored_file_patterns" : [
```
Comment (kkoskelin) [inline .nova/Configuration.json:2]: is this an editor config file?

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**